File: Test_Coverage/processed_dataset/tests/test_HumanEval_11.py
from functions.HumanEval_11 import string_xor

def test_HumanEval_11():
     assert string_xor('111000', '101010') == '010010'
     assert string_xor('1', '1') == '0'
     assert string_xor('0101', '0000') == '0101'

File: Test_Coverage_Java/src/test/java/test_HumanEval_11.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class HumanEval_11Test {

    @Test
    public void test_HumanEval_11() {
        assertEquals("010010", HumanEval_11.string_xor("111000", "101010"));
        assertEquals("0", HumanEval_11.string_xor("1", "1"));
        assertEquals("0101", HumanEval_11.string_xor("0101", "0000"));
    }
}